test/liblll/EndToEndTest.cpp

```diff
@@ -279,6 +279,26 @@ BOOST_AUTO_TEST_CASE(assembly_codecopy)
 	BOOST_CHECK(callFallback() == encodeArgs(string("abcdef")));
 }
 
+BOOST_AUTO_TEST_CASE(returndatasize_can_be_compiled)
+{
+	char const* sourceCode = R"(
+		(returnlll
```

Comment:
I'm not sure this is needed (though probably cannot hurt), because LLL just exposes every single opcode. Potentially we should test for every single one? :)